pages/[username].js

```diff
@@ -73,6 +73,16 @@ export default function User({ data }) {
         <div className="flex justify-center my-4">
           <ReactMarkdown>{data.bio}</ReactMarkdown>
         </div>
+        <div className="flex justify-center">
+          {
+            data?.tags?.length > 0 && 
+            data.tags.map((tag, index) => (
```

Comment:
I think it might be easier to read with something like...

```js
{ data.tags && data.tags.map(
```

Reply:
yes, but as no user have tags now in their profile we will get error
we can try this `{ data?.tags && data.tags.map(`

Reply:
`data` is always set, so there will not be an error